test/namenode/CacheTest.cc

```diff
@@ -0,0 +1,211 @@
+// Copyright 2017 Rice University, COMP 413 2017
+
+#include <iostream>
+#include "NameNodeTest.h"
+
+//TEST_F(NamenodeTest, cacheOneFile) {
```

Comment:
This should probably be uncommented once the ZKCONNECTIONLOSS error is addressed